frontend/src/App.css

```diff
@@ -70,3 +70,9 @@
 .xterm-screen {
   padding: 10px 0 0 10px;
 }
+
+@media screen and (max-width: 1000px) {
```

Comment:
Tailwind is installed on the frontend project, can we use that instead of this?

Reply:
Yes